addons/Lookup/Lookup.lua

```diff
@@ -0,0 +1,190 @@
+--[[
+	Copyright (c) 2018, Karuberu
```

Comment:
I copied it from the [Licensing section of the Writing Addons wiki page](http://github.com/Windower/Lua/wiki/Writing-Addons#licensing) and opted to use "THE COPYRIGHT HOLDERS OR CONTRIBUTORS" instead of my name in the last paragraph so that any other contributors will also be covered.

I'll change the copyright symbol and can change the last paragraph if required.

Reply:
Thanks for pointing it out, I adjusted the notice there. @alphaONE2 How do you feel about the change to the copyright notice?